packages/suite/src/components/settings/SettingsSection.tsx

```diff
@@ -51,16 +60,20 @@ export const SettingsSection = ({
                 {!title && customHeader}
                 {title && !customHeader && (
                     <Title>
-                        {icon && <StyledIcon icon={icon} size={16} color={theme.TYPE_LIGHT_GREY} />}
-                        {title}
+                        {icon && <Icon icon={icon} size={24} color={theme.iconDefault} />}
+                        <H3>{title}</H3>
                     </Title>
                 )}
                 {description && !customHeader && (
-                    <Description type="label">{description}</Description>
+                    <Description>
+                        <Paragraph type="label">{description}</Paragraph>
+                    </Description>
                 )}
             </Header>
 
-            <Card paddingType="none">{children}</Card>
+            <CardWrapper>
+                <Card paddingType="none">{children}</Card>
```

Comment:
please remove this `padding="none"`, it's mostly a legacy prop, and remve padding which have been replacing the card paddings in `SectionItem`. There is no need to have padding definitions elsewhere, `Card`s should have standardised paddings :)

Reply:
https://github.com/trezor/trezor-suite/pull/10050/commits/4e43da5f6ad6d8c577d90a35ae11cca64a284fe3

Reply:
Paddings in `SectionItem` are still there. Please remove the horizontal paddings and do not rented parring-top and padding-bottom for the first and last item respectively 🙏 Also I forgot to mention that the `Card` should have `paddingType="large"`, since it's quite big for `12px` paddings + they are larger in Figma _(even though the Card component with correct paddings is not used there)._